syntax = "proto3";

package LoginProtocol;

enum ID {
    ZERO                    = 0;

    C2S_LOGIN_BASE          = 1000;
    C2S_LOGIN_LOAD          = 1001;
    S2C_LOGIN_LOAD          = 1002;
    C2S_LOGIN_TOP           = 1099;
}
